README.md

```diff
@@ -52,6 +52,41 @@ The logs of the task can be inspected in real-time with:
 docker logs -f telemetryairflow_scheduler_1
 ```
 
+You can task logs and see cluster status on
+[the EMR console](https://us-west-2.console.aws.amazon.com/elasticmapreduce/home?region=us-west-2)
+
+By default, the results will end up in the `telemetry-test-bucket` in S3.
+If your desired task depends on other views, it will expect to be able to find those results
+in `telemetry-test-bucket` too. It's your responsibility to run the tasks in correct
+order of their dependencies.
+
+### Testing main_summary
+
+`main_summary` can be a good test case for any large changes to telemetry-batch-view,
+but you'll likely want to make some modifications before launching a test to keep runtime reasonable.
+
+Edit the `EMRSparkOperator` definition for the `main_summary` task in `dags/main_summary.py`
+to add some additional parameters to the `tbv_envvar` dictionary:
+
+```python
+        "channel": "nightly",   # run on smaller nightly data rather than release
+        "read-mode": "aligned", # group into fewer Spark tasks
```

Comment:
This is actually creating more RDDs than the default setting to accommodate a smaller dataset/cluster. For example, nightly may only read 5 partitions utilizing only 5/32 executors on the default "fixed" read-mode.